src/test/java/org/broadinstitute/hellbender/utils/mcmc/AdaptiveMetropolisSamplerUnitTest.java

```diff
@@ -0,0 +1,106 @@
+package org.broadinstitute.hellbender.utils.mcmc;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by davidben on 11/24/15.
+ */
+public class AdaptiveMetropolisSamplerUnitTest {
+    private static final double BETA_MIN = 0;
+    private static final double BETA_MAX = 1;
+    private static final double BETA_START = 0.5;
+    private static final double INITIAL_STEP_SIZE = 0.1;
+    private static final EmptyData EMPTY_DATA = new EmptyData();
+    private static final int NUM_BURN_IN_STEPS = 200;
+    private static final int NUM_SAMPLES = 500;
+
+
+    // factory for sampling the distribution Beta(a, b) defined on [0,1]
```

Comment:
if you add a static method for generating N samples to AdaptiveMetropolisSampler, can you write these tests without making GibbsSamplers (analogous to the test for SliceSampler)?  this would keep these two classes decoupled and make this test class much shorter.